.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 
 repos:
 -   repo: https://github.com/ambv/black
-    rev: stable
+    rev: 22.8.0
```

Comment:
Did a precommit autoupdate here, and I think it removed "stable". Could probably just undo this.